packages/storybook/src/executors/storybook/schema.json

```diff
@@ -50,19 +50,10 @@
     "staticDir": {
       "type": "array",
       "description": "Directory where to load static files from, array of strings",
-      "default": [],
       "items": {
         "type": "string"
       },
-      "x-deprecated": "This field has been renamed to staticDirs."
-    },
-    "staticDirs": {
-      "type": "array",
-      "description": "Directory where to load static files from, array of strings or object with from, to keys",
-      "default": [],
-      "items": {
-        "$ref": "#/definitions/staticDirPattern"
-      }
```

Comment:
If `staticDirs` is the replacement, do we definitely want to remove that from the schema?

Reply:
https://github.com/nrwl/nx/issues/9306#issuecomment-1066969343

Reply:
Let me know if you think I am missing something